### ISSUE TITLE
Remove build time dependencies from the Operator dist

### ISSUE THREAD

**Issue by vmuzikar** [keycloak/keycloak#22496]
### Description

The Operator distribution should not include any build time dependencies. This most probably includes the following:
```
io.fabric8:crd-generator-api:6.7.2
io.fabric8:crd-generator-apt:6.7.2
io.fabric8:generator-annotations:6.7.2
io.quarkiverse.operatorsdk:quarkus-operator-sdk-bundle-generator:6.2.1
io.sundr:builder-annotations:0.95.0
io.sundr:resourcecify-annotations:0.95.0
io.sundr:sundr-adapter-api:0.95.0
io.sundr:sundr-adapter-apt:0.95.0
io.sundr:sundr-adapter-reflect:0.95.0
io.sundr:sundr-codegen-api:0.95.0
io.sundr:sundr-codegen-apt:0.95.0
io.sundr:sundr-core:0.95.0
io.sundr:sundr-model-base:0.95.0
io.sundr:sundr-model-repo:0.95.0
io.sundr:sundr-model-utils:0.95.0
io.sundr:sundr-model:0.95.0
```

**Comment by shawkins** [comment]
io.quarkiverse.operatorsdk:quarkus-operator-sdk-bundle-generator:6.2.1 to be handled by quarkiverse/quarkus-operator-sdk#689

**Comment by metacosm** [comment]
Can you give https://github.com/quarkiverse/quarkus-operator-sdk/pull/690 a try to see if this solves the issue?

**Comment by shawkins** [comment]
@metacosm yes that fixes it

**Comment by metacosm** [comment]
> @metacosm yes that fixes it

Thank you, will merge and cut a release then!

**Comment by shawkins** [comment]
Sounds great.  That will also include the logging changes and is something we'll be able to align to in the near term?

**Comment by metacosm** [comment]
> Sounds great. That will also include the logging changes and is something we'll be able to align to in the near term?

Yes, 6.3.0 that just got released uses JOSDK 4.4.2 which contains the logging changes. Platform PR has been created as well: https://github.com/quarkusio/quarkus-platform/pull/931

**Comment by vmuzikar** [comment]
@metacosm Thanks for the quick fix. We'd need this in Quarkus 3.2 for RHBQ. Would a backport be possible?
